drivers/integration/docker/docker.go

```diff
@@ -25,8 +25,9 @@ type driver struct {
 }
 
 type volumeMapping struct {
-	Name             string `json:"Name"`
-	VolumeMountPoint string `json:"Mountpoint"`
+	Name             string                 `json:"Name"`
+	VolumeMountPoint string                 `json:"Mountpoint"`
+	VolumeStatus     map[string]interface{} `json:"Status"`
```

Comment:
What is `VolumeStatus`? More specifically, I know what the word `Status` means, but why is it represented by a map?